modules/imaging_qc/README.md

```diff
@@ -0,0 +1,36 @@
+## Imaging Quality Control Module
+
+#### Purpose
+The purpose of this module is to verify and help ensure imaging data has been uploaded into LORIS and reviewed for Quality Control (QC). It provides a queryable interface to check at a glance the completeness of imaging data collection for a study, and the status of QC review processes on them.  
+
+Note: this module is not the module in which visual/radiological QC is performed on scans, just the module in which QC assessments are *reviewed*. Links are included in this module so the user can jump to the Imaging Browser to mark QC assessments on images and visits.
+
+#### Intended Users
+This module is used by study administrators, imaging scientists, analysts, coordinators, and clinicians.
+
+#### Scope
+The Imaging QC module allows you to view specific data on uploaded image files, with many additional filtering options. The resulting table has a column for QC status, so you can see whether the upload has passed QC or not. The table provides a link to each file. You can download the table in `.csv` format. 
+
+Note: To QC a scan, the user must click the link leading to the Imaging Browser, and enter the QC details in that module.
+
+#### Configurations
+
+
+The `tblScanTypes` setting allows customization of which modalities are displayed in this module.
+
+> **caveat:** This module has a key dependency - the `mri_parameter_form` instrument must installed and properly configured. 
```

Comment:
This isn't a caveat, it's just another thing that needs to be configured and should be described as such.

Reply:
I would argue the caveat is definitely worth including because the module looks like it has a major bug otherwise, and a more elegant error message is not making it into this release (#5235).  

If this is the only issue holding this module documentation, can we merge and then lessen the caveat when #5235 is resolved?